project-management/src/main/java/life/qbic/projectmanagement/domain/model/project/purchase/Offer.java

```diff
@@ -0,0 +1,72 @@
+package life.qbic.projectmanagement.domain.model.project.purchase;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Lob;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.Random;
+
+/**
+ * <b>Offer</b>
+ *
+ * An offer in the context of project management, not finance and accounting.
+ *
+ * @since 1.0.0
+ */
+@Entity(name = "purchase_offer")
+public class Offer {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  private boolean signed;
+
+  private String fileName;
+
+  @Lob
```

Comment:
As this is a large object, maybe we should fetch it lazily as the JavaDoc implies.
```java
@Basic(fetch=LAZY)
```